addons/binding/org.openhab.binding.allplay/src/main/java/org/openhab/binding/allplay/internal/AllPlayBindingProperties.java

```diff
@@ -17,25 +17,34 @@
  */
 public class AllPlayBindingProperties {
 
-    private final Integer rewindSkipTimeInSec;
-    private final Integer fastForwardSkipTimeInSec;
+    private final double rewindSkipTimeInSec;
```

Comment:
@kaikreuzer, As I need to handle invalid string values anyway (if configured through cfg file), I have changed the implementation and created a new PR.
Btw. the property dictionary always gives me string values, even if set through PaperUI (in opposition to your other review comment). I wasn't able to get Integers in the config. Anyway, implementation should be on the safe side now.